pkg/service/catalog/api/types.go

```diff
@@ -0,0 +1,21 @@
+package api
+
+import (
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+type CatalogService struct {
+	// Target is a reference to a service
+	Target kapi.ObjectReference
+	// Type is a published service's claim type
+	Type string
+	// Description is a user-defined description of the published service
+	Description string
```

Comment:
The description for the service in the namespace is "something I exported
for my team" and the description in the catalog is a multi-paragraph intro
to the service including usage info.  Seems pretty different.  Exported
services are targeted at consumers.  Services in the UI are targeted at
namespace owners.

On Oct 2, 2015, at 2:18 PM, Andy Goldstein <notifications@github.com> wrote:

In pkg/service/catalog/api/types.go
<https://github.com/openshift/origin/pull/4822#discussion_r41051080>:

> @@ -0,0 +1,21 @@
> +package api
> +
> +import (
> +	kapi "k8s.io/kubernetes/pkg/api"
> +)
> +
> +type CatalogService struct {
> +	// Target is a reference to a service
> +	Target kapi.ObjectReference
> +	// Type is a published service's claim type
> +	Type string
> +	// Description is a user-defined description of the published service
> +	Description string

A counter argument is that can a catalog service have a different
description than the core service and the answer is yes

I would argue "no"

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/4822/files#r41051080>.